collect/collect.go

```diff
@@ -131,6 +135,13 @@ func InitChan(tsdbhost *url.URL, root string, ch chan *opentsdb.DataPoint) error
 		slock.Unlock()
 		return
 	})
+	Set("collect.discarded", Tags, func() (i interface{}) {
+		slock.Lock()
```

Comment:
Yes please. I'm going to merge this for now, but if you wanted to do that in a seperate pr it would be appreciated. Thanks!